examples/demo/src/routes/HomePage.tsx

```diff
@@ -1,10 +1,9 @@
 import algoliarecommend from '@algolia/recommend';
 import {
-  Recommend,
   TrendingFacets,
   TrendingItems,
   RecommendedForYou,
-} from '@algolia/recommend-react';
+} from '@algolia/recommend-react/dist/esm/experimental-personalization';
```

Comment:
alternatively if you put it directly in a `experimental-personalization` folder instead of inside dist/esm, that should work too, but again, same considerations apply.